src/main/java/io/github/hotelmanagement/model/price/Price.java

```diff
@@ -0,0 +1,17 @@
+package io.github.hotelmanagement.model.price;
+
+import io.github.hotelmanagement.model.room.RoomMapper;
+import io.github.hotelmanagement.model.room.RoomRepository;
+import lombok.ToString;
+
+@ToString
+public class Price {
+    private RoomMapper mapper;
+    private RoomRepository roomRepository;
+
+    public int countPrice(int beds){
+        // 1 bed - 80PLN
```

Comment:
można usunąć komentarz i tymaczasowo ustawić price: private static final int BED_PRICE = 80; potem można to pobrać z bazy